core/templates/domain/exploration/curated-exploration-validation.service.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to validate candidates for curated lessons
+ */
+
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+import { ExplorationSummaryBackendApiService, ExplorationSummaryBackendDict } from
+  'domain/summary/exploration-summary-backend-api.service';
+import { ReadOnlyExplorationBackendApiService, FetchExplorationBackendResponse } from
+  './read-only-exploration-backend-api.service';
+import { MultipleChoiceInputCustomizationArgsBackendDict } from
+  'extensions/interactions/customization-args-defs';
+import constants from 'assets/constants';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CuratedExplorationValidationService {
+  constructor(
+    private explorationSummaryBackendApiService:
+      ExplorationSummaryBackendApiService,
+    private readOnlyExplorationBackendApiService:
+      ReadOnlyExplorationBackendApiService) {}
+
+  /* *****************************************
+  Validate exploration settings.
+  ***************************************** */
```

Comment:
**Note:** This block of code was copied over from the original file, named `exploration-id-validation.service.ts`.